class/defaults.yml

```
@@ -1,5 +1,6 @@
 parameters:
   openshift4_keepalived:
-    namespace: openshift-operator-community
+    namespace: openshift-keepalived-operator
     channel: alpha
+    openshift_version: '${dynamic_facts:openshiftVersion:Major}.${dynamic_facts:openshiftVersion:Minor}'
```

Dynamic facts should not be used in a component, but only in VSHN specific parameter files. Further, please document this in `parameters.adoc`.

```suggestion
    openshift_version: '4.9'
```

---

Updated